### PR TITLE
small fix to begin with.

### PR DESCRIPTION
**Pull request by Vane11ope**
just fixed some small point concerned with that the screen tries to go up when there's no space to do that anymore.

### CODE REVIEW

**Comment by XVilka** [comment]
Can you please describe how to reproduce that bug you have fixed? Thank you.

File: libr/core/panels.c [hunk 1]
```diff
@@ -881,7 +881,9 @@ R_API int r_core_visual_panels(RCore *core) {
 			if (curnode == 1) {
 				r_core_cmd0 (core, "s-8");
 			} else {
-				panels[curnode].sy--;
+				if(panels[curnode].sy > 0) {
```
Comment (XVilka) [inline libr/core/panels.c]: please use the space between `if` and `(`

**Comment by Vane11ope** [comment]
how to reproduce the bug.

1. go to V! mode
2. push tab and move to symbols pane
3. keep pressing k to go up
4. try to go down with j
5. it will not go down until you push as much as extra pushes you did when you go up

File: libr/core/panels.c (hunk 1 above)
Comment (Vane11ope) [reply libr/core/panels.c]: i fixed this now.

**Comment by XVilka** [comment]
Good catch!

**Pull request merged**